packages/hash/README.md

```diff
@@ -282,7 +266,8 @@ The below `package.json` file outlines the minimum requirements a package has to
 }
 ```
 
-The above `devDependencies` are owed to our root eslint-config at `packages/hash/.eslintrc.json`.
+[//]: # (TODO: this path doesn't exist, repo root?)
```

Comment:
@kachkaev

Reply:
We should quick-fix this now to be correct, although experiments over in the BP repo (https://github.com/blockprotocol/blockprotocol/pull/551) may result in further changes, requiring this to be updated again soon.

Reply:
I don't get the meaning of these lines, let's delete them (269-272) 😅
Our setup will be somewhat like https://github.com/blockprotocol/blockprotocol/pull/551 in the end, as David has mentioned.

Reply:
IMHO we don't have to be too explicit about linters in the docs. Nobody would read this stuff anyway. Potential contributors know how to create a PR, which will be followed by CI checks. When linting fails, a contributor needs to know what to do, e.g.

https://github.com/hashintel/hash/blob/e26e373754e76b95b3598b50fd3b9efb9cce422e/.github/workflows/ci.yml#L17-L22

All other info is irrelevant.

Reply:
I have removed 269-272. Unsure if there were more changes being suggested in this thread